docs/scos/dev/glue-api-guides/202307.0/data-exchange-api/how-to-guides/how-to-configure-data-exchange-api.md

```diff
@@ -0,0 +1,132 @@
+---
+title: How to configure Data Exchange API endpoints.
+description: This guide shows how to configure the Data Exchange API endpoints.
+last_updated: June 23, 2023
+template: howto-guide-template
+redirect_from:
+  - /docs/scos/dev/glue-api-guides/202304.0/data-exchange-api/how-to-guides/how-to-configure-data-exchange-api.html
+---
+
+This document describes how to configure the Data Exchange API endpoints by executing SQL queries.
+
+To create an endpoint for interacting with entities in the database, you need to add a corresponding row to the `spy_dynamic_entity_configuration` table.
+
+The `spy_dynamic_entity_configuration` table contains the configuration of dynamic entity endpoints and has the following columns:
+
+| COLUMN | SPECIFICATION |
+| --- | --- |
+| id_dynamic_entity_configuration | The unique ID of the configuration. |
+| table_alias | An alias used in the request URL to refer to the endpoint. |
+| table_name | The name of the corresponding table in the database to operate on. |
+| is_active | Indicates whether the endpoint is enabled or disabled. |
+| definition | A JSON-formatted string containing the configuration details for each field in the table. |
+| created_at | Date and time when the configuration was created. |
+| updated_at | Date and time when the configuration was last updated. |
+
+Dive deeper into the configuration of the `spy_dynamic_entity_definition.definition` field.
```

Comment:
I don't understand how this field suddenly appeared here, there is not context. Also, it's not clear how the field is related to the following content

Reply:
`spy_dynamic_entity_configuration` schema fields are described right above in the table. `spy_dynamic_entity_definition.definition` column is mentioned as well

Reply:
Shouldn't it be `spy_dynamic_entity_configuration.definition` then?

Reply:
Oh, yes, it's a typo